### PR TITLE
[stable/rocketchat] upgrade to 1.2.3

### PR DESCRIPTION
**Pull request by verwilst**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Review by geekgonecrazy — APPROVED** [review]

File: stable/rocketchat/values.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 ## ref: https://hub.docker.com/r/rocketchat/rocket.chat/tags
 ##
 image:
-  repository: docker.io/library/rocket.chat
-  tag: 1.1.1
+  repository: rocketchat/rocket.chat
```
Comment (srueg) [inline stable/rocketchat/values.yaml]: Any objections to leaving the `docker.io/` part?
It's theoretically possible to configure the Docker deamon to use other registries than docker.io as default which might even pose a security issue...

**Review by srueg — CHANGES_REQUESTED** [review]

File: stable/rocketchat/values.yaml (hunk 1 above)
Comment (verwilst) [reply stable/rocketchat/values.yaml]: Done!

File: stable/rocketchat/values.yaml [hunk 2]
```diff
@@ -2,8 +2,8 @@
 ## ref: https://hub.docker.com/r/rocketchat/rocket.chat/tags
 ##
 image:
-  repository: docker.io/library/rocket.chat
-  tag: 1.1.1
+  repository: docker.io/rocketchat/rocket.chat
+  tag: 1.2.3
```
Comment (davidkarlsen) [inline stable/rocketchat/values.yaml:6]: please update the readme too

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

**Pull request merged**